### DISCUSSION TITLE
Can jsPsych be used mobile phone

### DISCUSSION THREAD

**Discussion by cestq** [jspsych/jsPsych#168] [Q&A] [answered]
same title

**Comment by jodeleeuw** [comment, marked as answer]
The short answer is yes.

The longer answer is that jsPsych's core features will all work on a mobile phone, but many of the plugins are designed for desktop use and won't work on mobile devices. You may need to modify plugins to get them to work on a mobile phone. For example, many plugins rely on keyboard presses but on many mobile devices the keyboard only appears when the focus is on a text input element.

**Reply by rgriscom** [reply]
Very late followup question: 
If a bluetooth keyboard is used with the mobile device, would that solve some of the plugin issues?

**Reply by jodeleeuw** [reply]
Yes, I think you could use the keyboard response plugins with a bluetooth keyboard.
